src/js/components/stock-movement-wizard/request/AddItemsPage.jsx

```diff
@@ -1072,7 +1072,7 @@ class AddItemsPage extends Component {
     if (this.state.values.statusCode === 'CREATED') {
       return apiClient.post(url, payload)
         .then(() => {
-          window.location = `/openboxes/stockMovement/list?type=REQUEST&movementNumber=${movementNumber}&submitted=true`;
+          window.location = `/openboxes/stockMovement/list??direction=INBOUND&movementNumber=${movementNumber}&submitted=true`;
```

Comment:
Remove extra ?

Reply:
@jmiranda I wanted to remove it but then I realized that once user submits request there is message displayed "Thank you for submitting your request. You can check the status of your request using stock movement number ...". So we need both params - movement number and submitted boolean.

Reply:
I assume you misinterpreted what I said. There’s no way you need two question marks (??) in the URL.

Reply:
@jmiranda Ohhh... Yes, I totally misinterpreted. Of course, you're right!

Reply:
@jmiranda Updated PR.